sample_data_config/ejp-xml-data-pipeline.config.yaml

```diff
@@ -0,0 +1,12 @@
+gcpProjectName: 'elife-data-pipeline'
+dataPipelineId: ejp_xml_pipeline_id
+dataset: '{ENV}'
+manuscriptTable: 'manuscript_{ENV}'
```

Comment:
why do we add the `_{ENV}` suffix to the tables when they are already in the `{ENV}` dataset?
Is this just to demonstrate that you could?

Reply:
quite frankly, i am not really bothered about the names of the tables in the config here which can be anything, but since you want the table names to be similar to the current table names, and you want to avoid the situation that a table being used by someone in an dataset gets truncated unintentionally, i decided to add anything that can mke the table name to be sufficiently different from any table name in the dataset